.env-devel

```diff
@@ -126,6 +126,7 @@ DIRECTOR_V2_TRACING={}
 # DYNAMIC_SCHEDULER ----
 DYNAMIC_SCHEDULER_LOGLEVEL=DEBUG
 DYNAMIC_SCHEDULER_PROFILING=1
+DYNAMIC_SCHEDULER_SCHEDULING_MODE=VIA_DIRECTOR_V2
```

Comment:
do you plan to have many other modes? if not I would go with a simpler boolean.

Reply:
Good point, I only want to support two modes. I will go with the boolean vlue